a8cteam51/team51-cli
# Added the github:update-repository-secret command

### Changes added
- Added the `github:update-repository-secret` command to update GH secrets

### Testing instructions
- Use the command to update a secret from a specific repo and confirm that it gets updated as expexted
- Use the command to update a secret from all our repos using the `--multiple=all` option (use `--verbose` to get a list of the repos that are getting updated)

### Examples
- `github:update-repository-secret  --verbose `
- `github:update-repository-secret  --verbose --multiple=all`
